rqd/deploy/opencue-rqd.service

```diff
@@ -8,12 +8,16 @@ Environment=OPTIONS=
 Environment=BIN=/usr/bin
 EnvironmentFile=-/etc/sysconfig/opencue-rqd
 ExecStart=/bin/bash -c "${BIN}/rqd ${OPTIONS}"
-ExecStop=/bin/bash -c "${BIN}/rqd --exit_now
+ExecStop=/bin/bash -c "${BIN}/rqd --exit_now"
 LimitNOFILE=500000
 LimitNPROC=500000
 StandardOutput=syslog+console
 StandardError=syslog+console
 SyslogIdentifier=rqd
 
+# Restart the service automatically if it crashes or exits
```

Comment:
Remove these lines. You don't need to document here something that's part of the systemd spec. 
```suggestion
```